TelCo.ColorCoder/PairNumbersFromColor.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using static TelCo.ColorCoder.Entity;
+
+namespace TelCo.ColorCoder
+{
+    class PairNumberFromColor
+    {
+        public static int GetPairNumberFromColor(ColorPair pair)
+        {
+            // Find the major color in the array and get the index
+            int majorIndex = -1;
+            for (int i = 0; i < colorMapMajor.Length; i++)
+            {
+                if (colorMapMajor[i] == pair.majorColor)
+                {
+                    majorIndex = i;
+                    break;
+                }
+            }
+
+            // Find the minor color in the array and get the index
+            int minorIndex = -1;
+            for (int i = 0; i < colorMapMinor.Length; i++)
```

Comment:
This `for` loop is almost a duplicate of the previous one. Any way to avoid this duplication?